read.py

```diff
@@ -1,67 +1,93 @@
-import string
-import json
 import pymongo
 from pymongo import MongoClient
+
 import config
-from datetime import datetime
-import dateutil.parser as dp
+from schemas import *
 
-def validate_user(db, token, email):
-    """
-    Returns the user if the authtoken provided as token is valid.
-    """
-    if not token or not email:
-        return False
+def tidy_results(res):
+    for i in res:
+        del i['_id']
+        del i['password']
```

Comment:
me like

Reply:
This is for JSON serialization - the two fields are some sort of raw byte streams that AWS doesn't serialize if we returned them.

Reply:
oh got it